tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -73,174 +73,174 @@
+                    friendly_name="amount of food stamp assistance/stamps", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="15", name='RECEIVES_SUB_CC', type='number', startIndex=41, endIndex=42,
-                    required=False, validators=[
+                    friendly_name="receives subsidized child care", required=False, validators=[
                         validators.isInLimits(0, 2),
                     ]),
               Field(item="16", name='AMT_SUB_CC', type='number', startIndex=42, endIndex=46,
-                    required=True, validators=[
+                    friendly_name="amount of subsidized child care", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="17", name='CHILD_SUPPORT_AMT', type='number', startIndex=46, endIndex=50,
-                    required=True, validators=[
+                    friendly_name="child support amount", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="18", name='FAMILY_CASH_RESOURCES', type='number', startIndex=50, endIndex=54,
-                    required=True, validators=[
+                    friendly_name="family cash resources", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="19A", name='CASH_AMOUNT', type='number', startIndex=54, endIndex=58,
-                    required=True, validators=[
+                    friendly_name="cash amount", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="19B", name='NBR_MONTHS', type='number', startIndex=58, endIndex=61,
-                    required=True, validators=[
+                    friendly_name="number of months", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="20A", name='CC_AMOUNT', type='number', startIndex=61, endIndex=65,
-                    required=True, validators=[
+                    friendly_name="child care amount", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="20B", name='CHILDREN_COVERED', type='number', startIndex=65, endIndex=67,
-                    required=True, validators=[
+                    friendly_name="children covered", required=True, validators=[
                         validators.isLargerThanOrEqualTo(0),
                     ]),
               Field(item="20C", name='CC_NBR_MONTHS', type='number', startIndex=67, endIndex=70,
-                    required=True, validators=[
+                    friendly_name="child care - number of months", required=True, validators=[
```

Comment:
another place where we add in a hyphen, but don't before other "number of months" below. Probably should be one way or another, but not both.